akka-actor/src/main/scala/akka/serialization/Serializer.scala

```diff
@@ -351,6 +351,7 @@ final case class DisabledJavaSerializer(system: ExtendedActorSystem) extends Ser
   }
 
   override def fromBinary(buf: ByteBuffer, manifest: String): AnyRef = {
+    log.warning(LogMarker.Security, "Incoming message is using DisabledJavaSerializer resulting in Message class None! Manifest was: [{}]", manifest)
     this.fromBinary(empty, None)
```

Comment:
This will result in double logging, I think it's better that this method logs and throws instead of delegating to the other `fromBinary`. 

Use a similar log message as in the other 
```
"Incoming message attempted to use Java Serialization even though `akka.actor.allow-java-serialization = off` was set! "
```